pwa/src/templates/templateParts/logsTable/LogsTableTemplate.module.css

```diff
@@ -0,0 +1,58 @@
+.container > *:not(:last-child) {
+  margin-block-end: var(--gateway-ui-size-lg);
+}
+
+.tableRow {
+  cursor: pointer;
+}
+
+.button > span > svg {
+  margin-inline-end: var(--gateway-ui-size-xs);
+}
+
+.message {
+  max-width: 200px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.message > :first-child {
+  width: 100%;
+}
+
+.DEBUG {
```

Comment:
We should rewrite these classnames to lowercase. (I know I made these classnames uppercase and I shouldn't have done that)